Packs/FreshDesk/Integrations/FreshDesk/FreshDesk.py

```diff
@@ -181,10 +181,7 @@ def reformat_ticket_context(context):
             context[new_key] = val
             del context[key]
         elif 'Cc' in key:
-            if key.endswith('s'):
-                new_key = key[:-1].replace('Cc', 'CC')
-            else:
-                new_key = key.replace('Cc', 'CC')
+            new_key = key[:-1].replace('Cc', 'CC') if key.endswith('s') else key.replace('Cc', 'CC')
```

Comment:
```suggestion
            new_key = (key[:-1] if key.endswith('s') else key)).replace('Cc', 'CC')
```

Reply:
This was an automatic change, there is so much i wanted to change but didnt because it would take too long but youve tempted me...